e2e/cases/svg/svgo.test.ts

```diff
@@ -51,6 +51,6 @@ test('should add id prefix after svgo minification', async () => {
   const content = readFileSync(indexJs!, 'utf-8');
 
   expect(
-    content.includes('"linearGradient",{id:"idPrefix_svg__a"}'),
+    content.includes('"linearGradient",{id:"idPrefix_svg__a"'),
```

Comment:
@svgr/webpack is compiled to React 16 JSX by default. Now Rsbuild uses SWC, so React new JSX will be used. The output are slightly different.